### ISSUE TITLE
[a11y] Popover

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/uikit#862]
1. Use attribute `aria-expanded="true"` for Popover when it opened
2. The Popover should be accessible via TAB key and expand after pressing Enter
3. The first element should be accessible after pressing Enter in Popover
4. One should have an opportunity to set custom `aria-label` for Popover

**Comment by Kyzyl-ool** [comment]
1. Is possible to use button as Popover element and pass all the necessary aria-* props on it
2. Was done in https://github.com/gravity-ui/uikit/pull/868
3. Also done in https://github.com/gravity-ui/uikit/pull/868
4. One can pass the prop to button as popover child element
